client/src/components/Layout/EventInfo/EventInfo.js

```diff
@@ -1,8 +1,77 @@
 import React, { Component } from 'react'
 import './eventInfo.css'
 import Background from '../../../Images/background.jpg'
+import ClockIcon from '../../../Images/icons8-clock-128.png'
+import CalendarIcon from '../../../Images/icons8-calendar-64.png'
+import LocationIcon from '../../../Images/icons8-location-96.png'
+import NoteIcon from '../../../Images/icons8-note-48.png'
+import PersonIcon from '../../../Images/icons8-person-60.png'
 
+const gapi = window.gapi
+const CLIENT_ID = '558072145685-hl8laqhvbqcd0f5vfrnt61v1ts1ls5lh.apps.googleusercontent.com'
+const API_KEY = 'AIzaSyBaXfE46xxRXpcw7vSUmIZaKlwJrec1bGY'
+const DISCOVERY_DOCS = ['https://www.googleapis.com/discovery/v1/apis/calendar/v3/rest']
+const SCOPES = 'https://www.googleapis.com/auth/calendar'
+
+const iconStyle={height : "2em" , width : "2em" , marginRight : "1em"}
 class EventInfo extends Component {
+  //handelClick will send event info to google calendar
+  handelClick=()=>{
```

Comment:
(almost) the entire method could be extracted to a different module so that it's reusable by other components as well and keeps this one clean. As I suggested in the previous sprint, it would be a good idea to have a module for all the API calls.

Reply:
:+1: